lib/JsonGenerator.js

```diff
@@ -6,11 +6,17 @@
 
 const { ConcatSource } = require("webpack-sources");
 
-const stringifySafe = data =>
-	JSON.stringify(data).replace(
+const stringifySafe = data => {
+	const stringified = JSON.stringify(data);
+	if (!stringified) {
+		return ""; // Invalid JSON
```

Comment:
return `"undefined"` here, elsewise this will result in invalid JS being generated.